### PR TITLE
#214: Status has a background colour & add other connectors

### PR DESCRIPTION
**Pull request by TomRadford**
Design:
image
Implementation:
image

### CODE REVIEW

File: src/components/Status.svelte [hunk 1]
```diff
@@ -1,21 +1,25 @@
 <script lang="ts">
-  export enum Levels {
-    SUCCESS = 'success',
-    WARNING = 'warning'
-  }
+  import { cn } from '@/utils/classname';
 
-  export let level:Levels = Levels.WARNING;
```
Comment (TomRadford) [inline src/components/Status.svelte:7]: removal of enums in favour of a literal union is just a personal choice to clean up this implementation a bit.

File: src/components/Connectors/Connectors.svelte [hunk 2]
```diff
@@ -1,52 +1,39 @@
...
-        {/if}
-      {/if}
-    </div>
+{#if isLoading}
+  Loading...
+{:else}
+  <div class="flex w-full flex-col gap-8 h-min">
+    <Connector
+      type="source"
+      items={$sourcesData.data.map((source) => ({ id: source.id.toString(), name: source.description, linked: !!source.active }))}
```
Comment (TomRadford) [inline src/components/Connectors/Connectors.svelte:27]: not sure if `active` is the right property to indicate if something is `linked`?

Comment (matthampton-stock2shop) [reply src/components/Connectors/Connectors.svelte:27]: I think so

File: src/components/Connectors/Connectors.svelte [hunk 3]
```diff
@@ -1,52 +1,39 @@
...
-            </tbody>
-          </Table>
-        {/if}
-      {/if}
-    </div>
+{#if isLoading}
+  Loading...
+{:else}
+  <div class="flex w-full flex-col gap-8 h-min">
+    <Connector
```
Comment (matthampton-stock2shop) [inline src/components/Connectors/Connectors.svelte:25]: Could we call this ConnectorGroup?

**Review by matthampton-stock2shop — CHANGES_REQUESTED** [review]

**Review by matthampton-stock2shop — APPROVED** [review]

**Pull request merged**